tests/multiInclude.c

```diff
@@ -1,46 +1,50 @@
 /*
-*  multiinclude test program
-*  validate that xxhash.h can be included multiple times and in any order
-*
-*  Copyright (C) Yann Collet 2013-present
```

Comment:
@Cyan4973 Unless you are a time traveler, I believe this should be 2020-present, no?

Reply:
yep ! I guess it was part of the copy paste